Mapbox/MapboxMapsGestures/GestureManager.swift

```diff
@@ -283,7 +283,10 @@ extension GestureManager: GestureHandlerDelegate {
 
     // Pan has ended on the MapView with a residual `offset`
     internal func panEnded(with offset: CGPoint) {
-        self.cameraManager.moveCamera(by: offset, animated: true)
+        if let pitch = self.cameraManager.mapView?.pitch,
+           pitch == 0.0 {
+            self.cameraManager.moveCamera(by: offset, animated: true)
```

Comment:
```suggestion
            cameraManager.moveCamera(by: offset, animated: true)
```

Reply:
we are using `self.` in most places so would like to keep that for consistency

Reply:
Sounds good.